### ISSUE TITLE
Investigate whether we can use systemd Path Units

### ISSUE THREAD

**Issue by probonopd** [AppImage/appimaged#8]
Investigate whether we can use systemd Path Units rather than using `inotify` ourselves. This would simplify the tool, make it more flexible (could be triggered in different ways), and would even make the search paths user-configurable (through systemd).

https://www.linuxjournal.com/content/linux-filesystem-events-inotify (search for "Path Units under systemd").

**Comment by TheAssassin** [comment]
Did you read that if systemd is run as PID 1, the functionality is limited? I just had a look, and Ubuntu for instance runs `/sbin/init` as PID 1.

**Comment by probonopd** [comment]
Yes, I read this, hence we need to _investigate_ whether it is suitable as a solution. I don't know yet.
